src/neuroglancer/segmentation_display_state/base.ts

```diff
@@ -17,18 +17,41 @@
 import {SharedDisjointUint64Sets} from 'neuroglancer/shared_disjoint_sets';
 import {Uint64Set} from 'neuroglancer/uint64_set';
 import {Uint64} from 'neuroglancer/util/uint64';
+import {vec3} from 'neuroglancer/util/geom';
+import {SharedTrackableValue} from 'neuroglancer/shared_trackable_value';
+
+export interface Bounds {
+  center: vec3;
+  size: vec3;
+};
+
+export class ClipBounds extends SharedTrackableValue<Bounds> {
+  toJSON(): {}|null {
+    let bounds = this.value;
+    if (bounds === undefined) { 
+      return null;
```

Comment:
Returning undefined instead of null may be more convenient.  Because `JSON.stringify({clipBounds: undefined}) === '{}'`, you can simplify the code that calls this method.